twitter.html

```diff
@@ -0,0 +1,86 @@
+</head>
+
+<body>
+    <main class="container">
+        <div class="left">
+            <div class="inputBoxes">
+                <form>
+                    <input type="text" class="login_info" placeholder="Phone, Email or username">
+                    <input type="text" class="password" placeholder="Password">
+                    <button class="logInButton" type="button">Log in</button>
+                </form>
+            </div>
+            <div class="bird">
+                <img class="birdImg" src="/Users/marvinthompson/Desktop/UNIT2/Pursuit-Core-Web-Twitter-CSS-Assignment/images/birdlogo.png" alt="Twitter logo">
+            </div>
+            <div class="information">
+                <h1>See what's happening in the world right now.</h1>
+            </div>
+            <div class="miscInfo">
+
+                <h3>Join Twitter today.</h3>
+            </div>
+            <div class="buttons1">
+                <button class="SignUp" type="button">Sign Up</button>
+                <section>
+
+            </div>
+            <div class="buttons2">
+
+                    <button class="LogIn2" type="button">Log in</button>
+                </section>
+            </div>
+        </div>
+
+        <div class="right">
+            <div class="content">
+                <ul>
+                    <li>Follow your interests.</li>
+                    <br>
+                    <li>Hear what people are talking about.</li>
+                    <br>
+                    <li>Join the conversation.</li>
+                </ul>
+                
+            </div>
+        </div>
+
+        
+</main>
+                <div class="footerDiv">
```

Comment:
this div seems unnecessary. The `<footer>` in itself is a div already. It contains the `<ul>` that contains the `<li>`s. And you didn't close that div.